packages/phone-number-privacy/common/src/interfaces/domains.ts

```diff
@@ -0,0 +1,26 @@
+export interface SequentialDelayDomain {
+  name: 'Sequential Delay Domain'
```

Comment:
Since the name should be unique among all EIP-712 use cases, I'd recommend making the `name` field start with `ODIS`
```suggestion
  name: 'ODIS Sequential Delay Domain'
```